README.md

```diff
@@ -480,6 +486,10 @@ IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2022. [YouTube](https
 - <a name="zhu2022learning"></a>Zhu, Z., Hou, J., Lyu, X.,  
 *[Learning Graph-embedded Key-event Back-tracing for Object Tracking in Event Clouds](https://openreview.net/pdf?id=hTxYJAKY85)*,  
 Thirty-sixth Conference on Neural Information Processing Systems (NeurIPS), 2022. [PDF](https://openreview.net/pdf?id=hTxYJAKY85), [code](https://github.com/ZHU-Zhiyu/Event-tracking).
+- <a name="messikommer23cvpr"></a>Nico Messikommer*, Carter Fang*, Mathias Gehrig, Davide Scaramuzza,  
```

Comment:
So far, we have not used asterisks to denote contribution of the co-authors. Think whether you want to have it (all entries should be revised).